inc/dashboard/namespace.php

```diff
@@ -0,0 +1,436 @@
+	$microcopy = [
+		'block' => __( 'List of XBs modified in the selected date range', 'altis-analytics' ),
+		'views' => __( 'Total unique views of the XBs during the selected date range', 'altis-analytics' ),
+		'conversion' => __( 'Average conversion is calculated as the total conversion expressed as a % of total unique views of the XB during the selected date range', 'altis-analytics' ),
+		'details' => __( 'Block last modified date and author', 'altis-analytics' ),
+	];
+
+	foreach ( $xb_columns as $column => $title ) {
+		$xb_columns[ $column ] = '<span title="' . esc_html( $microcopy[ $column ] ) . '">' . esc_html( $title ) . '</span>
+			<span class="screen-reader-text">' . esc_html( $microcopy[ $column ] ) . '</span>';
+	}
+
+	return $xb_columns;
+}
+
+/**
+ * Update the Block column's orderby value.
+ *
+ * This displays the title, but the column is a custom column called "block", so we need to change the orderby value to "title".
+ *
+ * @param array $vars An array of query variables passed in from the filter.
+ *
+ * @return array The updated query variables.
+ */
+function xb_block_column_orderby( $vars ) : array {
+	if ( isset( $vars['orderby'] ) && 'block' === $vars['orderby'] ) {
+		$vars['orderby'] = 'title';
+	}
+
+	return $vars;
+}
+
+/**
+ * Render links for date range data.
+ */
+function render_date_range_links() {
+	$ranges = [
+		[
+			'days' => 7,
+			'label' => __( '7 days', 'altis-analytics' ),
+		],
+		[
+			'days' => 30,
+			'label' => __( '30 days', 'altis-analytics' ),
+		],
+		[
+			'days' => 90,
+			'label' => __( '90 days', 'altis-analytics' ),
+		],
+	];
+
+	$days = get_days_view();
+	?>
+	<div class="xb-analytics-date-range">
+		<?php
+		foreach ( $ranges as $current ) {
+			// Compare the current days value with the current days value pulled from the url query variables.
+			if ( $days === $current['days'] ) {
+				$selected = 'selected';
+			} else {
+				$selected = '';
+			}
+			?>
+			<a href="<?php echo esc_url_raw( add_query_arg( [ 'days' => $current['days'] ] ) ); ?>" class="<?php echo esc_html( "date-range-button $selected" ); ?>"><?php echo esc_html( $current['label'] ); ?></a>
+		<?php } ?>
+	</div>
+	<?php
+}
+
+/**
+ * Render the block name column data.
+ */
+function render_block_column() {
+	global $post;
+	?>
+	<strong><a href="<?php echo esc_url_raw( admin_url( '/admin.php?page=xb-analytics&post=' . $post->ID ) ); ?>"><?php echo esc_attr( $post->post_title ); ?></a></strong>
+	<?php
+}
+
+/**
+ * Render the last modified / author column data.
+ */
+function render_last_modified_author() {
+	global $post;
+	?>
+	<div class="post--last-modified">
+		<span class="post--last-modified-date"><?php echo esc_attr( get_the_modified_date( '', $post ) ); ?></span>
+		<span class="post--last-modified-time"><?php echo esc_attr( get_the_modified_time( '', $post ) ); ?></span>
+	</div>
+	<div class="post--author">
+		<?php echo esc_attr( get_the_author_meta( 'display_name', $post->post_author ) ); ?>
```

Comment:
Posted a question on the issue but what're your thoughts on this juxtaposition of data? Does it make sense to you that these are combined?

Reply:
Based on that convo, and the idea that these might need to be sortable as well, let's split them into separate columns.